internal/app/storage/storage.go

```diff
@@ -0,0 +1,35 @@
+package storage
+
+//type ObjStorage struct {
+//	shortUrl string
+//}
+
+type Storage map[string]string
+
+func NewStorage() Storage {
+	arr := make(Storage)
+	return arr
+}
+
+func (strg Storage) Save(newUrl string, oldUrl string) string {
```

Comment:
ресивер лучше делать по ссылке, если не нужно по каким-то причинам создавать копию структуры
```suggestion
func (s *Storage) Save(newUrl string, oldUrl string) string {
```